docs/en/04-supported-codebases/mmdet3d.md

```diff
@@ -1,33 +1,99 @@
 # MMDetection3d Deployment
 
-- [Installation](#installation)
-  - [Install mmdet3d](#install-mmdet3d)
-  - [Install mmdeploy](#install-mmdeploy)
+- [Install mmdet3d](#install-mmdet3d)
 - [Convert model](#convert-model)
-- [Model specification](#model-specification)
 - [Model inference](#model-inference)
-  - [Backend model inference](#backend-model-inference)
-  - [SDK model inference](#sdk-model-inference)
 - [Supported models](#supported-models)
 
 ______________________________________________________________________
 
 [MMDetection3d](https://github.com/open-mmlab/mmdetection3d) aka `mmdet3d` is an open source object detection toolbox based on PyTorch, towards the next-generation platform for general 3D detection. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
 
-## Installation
+## Install mmdet3d
 
-### Install mmdet3d
+These branches are required for mmdet3d deployment
 
-### Install mmdeploy
+| codebase |  commit   |
+| :------: | :-------: |
+| mmdet3d  | v1.1.0rc1 |
+|   mmcv   | v2.0.0rc1 |
+|  mmdet   | v3.0.0rc1 |
+|  mmseg   | v1.0.0rc0 |
+
+First checkout and install mmcv/mmdet/mmseg/mmdet3d
+
+```bash
+git clone https://github.com/open-mmlab/mmcv  --branch v2.0.0rc1
+cd mmcv
+python3 -m pip install .
+cd -
+
+git clone https://github.com/open-mmlab/mmdetection  --branch v3.0.0rc1
+cd mmdetection
+python3 -m pip install .
+cd -
+
+git clone https://github.com/open-mmlab/mmsegmentation  --branch v1.0.0rc0
+cd mmsegmentation
+python3 -m pip install .
+cd -
+
+git clone https://github.com/open-mmlab/mmdetection3d --branch v1.1.0rc1
+cd mmdetection3d
+python3 -m pip install .
+cd -
+```
+
+After installation, `tools/check_env.py` should display mmdet3d version normally
+
+```bash
+python3 tools/check_env.py
+..
+11/11 13:56:19 - mmengine - INFO - **********Codebase information**********
+11/11 13:56:19 - mmengine - INFO - mmdet:       3.0.0rc1
+11/11 13:56:19 - mmengine - INFO - mmseg:       1.0.0rc0
+..
+11/11 13:56:19 - mmengine - INFO - mmdet3d:     1.1.0rc1
+```
 
 ## Convert model
 
-## Model specification
+For example, use `tools/deploy.py` to convert centerpoint to onnxruntime format
+
+```bash
```

Comment:
use mim to dowload config and ckpt

Reply:
mim not support mmdet3d=1.1.0rc1 and mmdet3d dev3d-1.x has changed config name.